src/components/Icon/Icon.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+export interface IconProps {
+  color?: string;
+  size?: string | number;
+}
+
+const Icon: React.FC<IconProps> = ({ size = 24, color = 'currentColor' }: IconProps) => {
+  return (
+    <div>
+      {size}
+      {color}
+    </div>
+  );
```

Comment:
It was confusing to me what this component was doing. I figured that it is necessary for Docz to inherit the props. 

The component can just return `<></>`. Also lets add a comment so it's clear that this component is only ment for Docz.

Reply:
Fine for me! Had it that way initially but Docz was throwing errors. In hindsight, the errors were probably for something else